examples/tac_deploy/build.sh

```diff
@@ -10,9 +10,9 @@ echo USE_CLIENT $USE_CLIENT
 mkdir /data
 
 # setup the agent
-aea fetch fetchai/tac_controller:latest
+aea fetch --local fetchai/tac_controller:latest
```

Comment:
Don;t commit the flag please; just use it locally; or add an env variable to set the flag (default should be remote)